purescript.cabal

```diff
@@ -2,7 +2,7 @@ cabal-version:  2.4
 
 name:           purescript
 -- note: When updating the prerelease identifier, update it in app/Version.hs too!
-version:        0.15.0-rc1
+version:        0.15.0
```

Comment:
Without this change, I get the following error:
```
$ stack build
Cabal file warning in/home/jordan/Programming/Projects/purescript/purescript.cabal@5:27: version with tags

Reply:
Isn't it just a warning? AFAIK we've had no trouble with this [in the past](https://travis-ci.org/github/purescript/purescript/jobs/751718020#L582).

Reply:
Ah, you're right. When I saw that `stack build` didn't produce any output, I thought it meant the build failed. But it didn't produce output because I had run `stack build` previously and there was nothing else to do.

I've reverted the commit.